### ISSUE TITLE
ContainerSet's containersEmitter

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/fractions-intro#61]
What is it? I'm not sure from the name, and it doesn't have documentation.

**Comment by michaelm396** [comment]
The containersEmitter, first initialized in ContainerSet, is a public object which signals once a container and/or cell has been updated. This is important because it signals to the Screen View an update to the fill property of a container or cell. I have refactored the name of the object to 'updatedContainersEmitter' in order to more closely resemble its functionality. I have also added documentation to ContainerSet that further describes the function of the object. Im assigning this issue back to @jonathanolson for further review. Thanks. (updated push: 2e527ed91a9b5fdaca570cfb157a5e5995355b3d)
